rfd/0086/compat.md

```diff
@@ -0,0 +1,7 @@
+## Backwards compatibility and support
+
+The changes in this RFD are not backwards compatible with ContainerPilot 2.x configuration. For end users using Consul as their discovery backend, each container image can be safely updated independently; there is no communication between ContainerPilot instances except through the Consul API.
```

Comment:
Updated in https://github.com/joyent/rfd/pull/13/commits/c361026cb18036c23d5be8e17a7eb13a0407d983